dataikuapi/dss/admin.py

```diff
@@ -946,3 +946,24 @@ def get_raw(self):
         Gets the whole status as a raw dictionary.
         """
         return self.status
+
+
+class DSSInstanceVariables(dict):
+    """
+    Dict containing the instance variables. The variables can be modified directly in the dict and persisted using its `save` method.
+
+    Do not create this directly, use :meth:`dataikuapi.dssclient.DSSClient.get_global_variables`
```

Comment:
While checking the doc PR, I noticed that this method link needs to be `dataikuapi.DSSClient.get_global_variables`